canonical-web-and-design/vanilla-framework
# Add/serve JSON data for toast nofitications

## Done

- Added json file for toast notifications
- Add route from app.py. I'm not sure how to serve all `.json` files in the directory, so for the time being have just served the toast notifications file

Fixes [#1203](https://github.com/canonical-web-and-design/vanilla-squad/issues/1203)

## QA

- Open [demo](https://vanilla-framework-4315.demos.haus/)
- go to https://vanilla-framework-4315.demos.haus/docs/examples/patterns/notifications/toast.json and check the json data exists

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
